rllib/env/tests/test_multi_agent_episode.py

```diff
@@ -2565,32 +2565,28 @@ def test_cut(self):
 
     def test_slice(self):
         # Generate a simple multi-agent episode.
-        observations = [
-            {"a0": 0, "a1": 0},
-            {"a1": 1},
-            {"a1": 2},
-            {"a0": 3, "a1": 3},
-            {"a0": 4},
-            {"a0": 5, "a1": 5},
-            {"a0": 6, "a1": 6},
-            {"a1": 7},
-            {"a1": 8},
-            {"a0": 9},
-        ]
-        actions = observations[:-1]
-        rewards = [{aid: r / 10 + 0.1 for aid, r in a.items()} for a in actions]
-        episode = MultiAgentEpisode(
-            observations=observations,
-            actions=actions,
-            rewards=rewards,
-            len_lookback_buffer=0,
+        episode = self._create_simple_episode(
+            [
+                {"a0": 0, "a1": 0},
+                {"a1": 1},
+                {"a1": 2},
+                {"a0": 3, "a1": 3},
+                {"a0": 4},
+                {"a0": 5, "a1": 5},
+                {"a0": 6, "a1": 6},
+                {"a1": 7},
+                {"a1": 8},
+                {"a0": 9},
+            ]
         )
         check(len(episode), 9)
 
         # Slice the episode in different ways and check results.
         # Empty slice.
         slice_ = episode[100:100]
```

Comment:
You mean to return a new `InfiniteLookbackBuffer` instead?
Yeah, that could be an option, too. I'm not sure. If we can safely change that, it would be better, I think. The good thing is that this API is not user-facing, so we can still change it later.